Client/Helpers/MenuBuilderHelper.swift

```diff
@@ -48,7 +48,8 @@ class MenuBuilderHelper {
             UIKeyCommand(title: .KeyboardShortcuts.ZoomIn, action: #selector(BrowserViewController.zoomIn), input: "+", modifierFlags: .command, discoverabilityTitle: .KeyboardShortcuts.ZoomIn),
             UIKeyCommand(title: .KeyboardShortcuts.ZoomOut, action: #selector(BrowserViewController.zoomOut), input: "-", modifierFlags: .command, discoverabilityTitle: .KeyboardShortcuts.ZoomOut),
             UIKeyCommand(title: .KeyboardShortcuts.ActualSize, action: #selector(BrowserViewController.resetZoom), input: "0", modifierFlags: .command, discoverabilityTitle: .KeyboardShortcuts.ActualSize),
-            UIKeyCommand(title: .KeyboardShortcuts.ReloadPage, action: #selector(BrowserViewController.reloadTabKeyCommand), input: "r", modifierFlags: .command, discoverabilityTitle: .KeyboardShortcuts.ReloadPage)
+            UIKeyCommand(title: .KeyboardShortcuts.ReloadPage, action: #selector(BrowserViewController.reloadTabKeyCommand), input: "r", modifierFlags: .command, discoverabilityTitle: .KeyboardShortcuts.ReloadPage),
+			UIKeyCommand(title: .KeyboardShortcuts.ReloadWithoutCache, action: #selector(BrowserViewController.reloadTabIgnoringCacheKeyCommand), input: "r", modifierFlags: [.command, .shift], discoverabilityTitle: .KeyboardShortcuts.ReloadWithoutCache)
```

Comment:
The task is asking for the shortcut to be per Desktop, which is CTRL+F5. This means input and modifierFlags need to be adjusted here.

Reply:
@lmarceau apologies, The task was a little confusing having changed from the initial message down the comments. I have made the change as per your message above, using "CTRL+F5". However, the F5 input key code is only available since iOS 13.4, so anyone running a version less than that won't have the shortcut available. That shouldn't be too much of a problem though as the app target seems to be 13 so should not be affecting too many people.

Reply:
Then we can add this shortcut for iOS > 13.4, we'll eventually remove iOS 13 support and we'll be able to clean this up then.